<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxIntMapSettings">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>140</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QToolButton" name="toolButtonIntMapSettingsSave">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>160</y>
        <width>115</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>Save and Exit</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelIntMapSettingsHeadline">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>200</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lato</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Interactive Map</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButtonIntMapSettingsCancel">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>160</y>
        <width>115</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelIntMapSettingsMapSetting">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lato</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Map Setting:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelIntMapSettingsMapHeight">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lato</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Map Height [m]:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelIntMapSettingsMapWidth">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>110</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lato</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Map Width[m]:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditIntMapSettingsHeight">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>105</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditIntMapSettingsWidth">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>105</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxIntMapSettingsDefault">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Default</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxIntMapSettingsLaboratory">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>70</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Laboratory</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
